frontend-react/src/components/FileHandlers/FileHandler.tsx

```diff
@@ -276,65 +328,52 @@ function FileHandler() {
                     <p className="margin-top-05">{fileName}</p>
                 </>
             )}
-            {isFileSuccess && warnings.length === 0 && (
-                <FileSuccessDisplay
-                    extendedMetadata={{
-                        destinations,
-                        timestamp: successTimestamp,
-                        reportId,
-                    }}
-                    heading="Validate another file"
-                    message={successDescription}
-                    showExtendedMetadata={false}
-                />
-            )}
-            {warnings.length > 0 && (
-                <RequestedChangesDisplay
-                    title={RequestLevel.WARNING}
-                    data={warnings}
-                    message="The following warnings were returned while processing your file. We recommend addressing warnings to enhance clarity."
-                    heading="File validated with recommended edits"
-                />
-            )}
-            {errors.length > 0 && (
-                <RequestedChangesDisplay
-                    title={RequestLevel.ERROR}
-                    data={errors}
-                    message={errorMessaging.message}
-                    heading={errorMessaging.heading}
-                />
-            )}
-            {hasQualityFilterMessages && (
-                <FileQualityFilterDisplay
-                    destinations={qualityFilterMessages}
-                    heading=""
-                    message={`The file does not meet the jurisdiction's schema. Please resolve the errors below.`}
+
+            {fileHandlerStep === FileHandlerSteps.STEP_ONE && (
```

Comment:
I wonder if we could simplify toggling the currently shown component like (`CurrentStep = FileHandlerStepOne|FileHandlerStepTwo|...`).  Then we could always pass the total set of usable props:

```tsx
<CurrentStep
    fileType={ ... }
    handleNextFileHandlerStep={ ... }
    onSchemaChange={ ... }
    ...otherStuff
/>
```

These steps are mutually exclusive so we'd only have to render one at a time, and it'd be easier to update one place for all that in my opinion 🤷 

OR we could even do something like come up with a common approach to multistep flows like this:
```tsx
function getOrderedSteps(state) {
    return [{
        Component: FileHandlerStepOne,
        // whether or not the user is able to proceed to the next step
        isValid: Boolean(state.selectedSchemaOption)
        // whether or not the step should be rendered
        isShown: !state.selectedSchemaOption,
        // additional props passed to this step's component
        additionalProps: { ... }
    }, 
    ...
    ]
}
```